integration-tests/src/tests/client/cold_storage.rs

```diff
@@ -201,24 +202,26 @@ fn test_cold_db_head_update() {
     genesis.config.epoch_length = epoch_length;
     let mut chain_genesis = ChainGenesis::test();
     chain_genesis.epoch_length = epoch_length;
-    let mut env = TestEnv::builder(chain_genesis)
-        .runtime_adapters(create_nightshade_runtimes(&genesis, 1))
-        .build();
-
     let store = create_test_node_storage_with_cold();
+    let hot_store = &store.get_store(Temperature::Hot);
+    let cold_store = &store.get_store(Temperature::Cold);
+    let runtime_adapter = create_nightshade_runtime_with_store(&genesis, &hot_store);
+    let mut env = TestEnv::builder(chain_genesis).runtime_adapters(vec![runtime_adapter]).build();
 
     for h in 1..max_height {
         env.produce_block(0, h);
         update_cold_head(&*store.cold_db().unwrap(), &env.clients[0].runtime_adapter.store(), &h)
             .unwrap();
 
-        assert_eq!(
-            &store.get_store(Temperature::Cold).get_ser::<Tip>(DBCol::BlockMisc, HEAD_KEY).unwrap(),
-            &env.clients[0]
-                .runtime_adapter
-                .store()
-                .get_ser::<Tip>(DBCol::BlockMisc, HEAD_KEY)
-                .unwrap()
-        );
+        let head = &env.clients[0]
+            .runtime_adapter
+            .store()
+            .get_ser::<Tip>(DBCol::BlockMisc, HEAD_KEY)
+            .unwrap();
+        let cold_head_in_hot = hot_store.get_ser::<Tip>(DBCol::BlockMisc, COLD_HEAD_KEY).unwrap();
+        let cold_head_in_cold = cold_store.get_ser::<Tip>(DBCol::BlockMisc, HEAD_KEY).unwrap();
+
+        assert_eq!(head, &cold_head_in_cold, "checking cold_head_in_cold");
```

Comment:
The assert error message is tautological. The assertion by itself reports that `cold_head_in_cold` is different from `head`. The error message doesn't add anything new.